### PR TITLE
Don't allow page to be scrolled while hinting

### PR DESCRIPTION
**Pull request by jebby**
Currently while in hint mode, if the user scrolls with the mouse, the hint labels are left on the screen at the original position when the user started hinting at.
This makes it a little hard to follow a hint accurately if you accidentally scroll.

For instance, my touchpad is extremely sensitive and picks up accidental scroll events from time to time when my palm touches the edge. This makes hinting extremely frustrating at times because I have to restart the hint sequence after scrolling back to the position I originally started hinting at.

If this is not the best place to implement this change, let me know.

### CODE REVIEW

File: qutebrowser/browser/eventfilter.py [hunk 1]
```diff
@@ -162,6 +162,12 @@ def _handle_wheel(self, e):
...
             self._ignore_wheel_event = False
             return True
+
+        # Don't allow scrolling while hinting
+        mode = modeman.instance(self._tab.win_id).mode
+        if mode == usertypes.KeyMode.hint:
+            return True
+
         elif e.modifiers() & Qt.ControlModifier:
             mode = modeman.instance(self._tab.win_id).mode
```
Comment (The-Compiler) [inline qutebrowser/browser/eventfilter.py:172]: No need to query this again here now that it's already done above.

**Comment by jebby** [comment]
Nice catch! I meant to remove that line before committing, but I guess it had slipped my mind before I pushed. I have removed the extra query.

**Comment by The-Compiler** [comment]
If pylint/flake8 fail, that's my fault - I messed that up in master, should be fixed in a bit. Thanks for being the guinea pig for those automation tests, and sorry for the noise!

**Comment by jebby** [comment]
It's not a problem. I'm just glad to contribute and hope that it helps others as much as it helps me.

**Comment by The-Compiler** [comment]
Merged, thanks! :+1: Definitely seems like a good idea as long as hints don't scroll with the page.
